ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -386,14 +386,15 @@ Resources:
             01_tag_root_disk:
               command: !Join
                 - ''
-                - - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) "
+                - - "set -x;"
+                  - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) &&"
                   - "ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes --region $AWS_REGION "
                   - "--filters Name=attachment.instance-id,Values=$EC2_INSTANCE_ID "
-                  - "--query Volumes[].VolumeId --out text); "
-                  - "EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION ec2 describe-tags --filters 'Name=resource-id,Values=$EC2_INSTANCE_ID') "
-                  - "PROJECT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Project\").Value ') "
-                  - "DEPARTMENT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Department\").Value ') "
-                  - "OWNER_EMAIL=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"OwnerEmail\").Value ') "
+                  - "--query Volumes[].VolumeId --out text) &&"
```

Comment:
`&&` doesn't seem necessary here?

Reply:
All changes made have been tested and were necessary